website/docs/docs/faq.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 5
+title: FAQ
+---
+
+# Frequently asked questions
+
+**What is Sandpack?**
+
+Sandpack is an open ecosystem of components and utilities that allow you to compile and run modern frameworks in the browser. You can either use one of our predefined `components` for embedding the *CodeSandbox* experience into your projects, or you can build your own version of `sandpack`, on top of our standard components and utilities. As you walk through this guide, you will get deeper into our ecosystem.
+
+**How to load local or private dependencies?**
+
+Currently, Sandpack doesn’t have a way to consume private dependencies from any kind of registry service, due to the bundler host is shared with all Sandpack consumers apps. However, you can pass local dependencies just like a regular file or using external resource API:
```

Comment:
```suggestion
Currently, Sandpack doesn’t have a way to consume private dependencies from any kind of registry service, because the bundler host is shared with all Sandpack consumers apps. However, you can pass local dependencies just like a regular file or using the external resource API:
```